banzai_floyds/orders.py

```diff
@@ -52,29 +49,30 @@ def smooth_order_weights(params, x, height):
     # Note the minus sign for the top of the filter. This flips the logistic function. This means that x0 also has to
     # flip signs. We also have to add a half to each side of the filter so that the edges are at the edges of pixels
     # as the center of the pixels are the integer coordinates
-    weights = logistic(centered_coordinates, x0=-(height // 2 + 0.5), k=2)
-    weights *= logistic(-centered_coordinates, x0=-(height // 2 + 0.5), k=2)
+    k = 2
+    half_height = height // 2 + 0.5
+    weights = expit(k * (y2d - y_centers + half_height))
+    weights *= expit(k * (-y2d + y_centers + half_height))
     return weights
 
 
 def smooth_order_jacobian(theta, x, i, height, k=2):
```

Comment:
It would help me understand better what's going on if there was documentation for the arguments here